src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -4704,6 +4704,25 @@ Note* MusicXMLParserPass2::note(const QString& partId,
         setNoteHead(note, noteheadColor, noteheadParentheses, noteheadFilled);
         note->setVisible(hasHead && printObject);     // TODO also set the stem to invisible
 
+        if (mnd.calculatedDuration().isValid()
+            && mnd.specifiedDuration().isValid()
+            && mnd.calculatedDuration().isNotZero()
+            && mnd.calculatedDuration() != mnd.specifiedDuration()) {
+            // convert duration into note length
+            Fraction durationMult { (mnd.specifiedDuration() / mnd.calculatedDuration()).reduced() };
+            durationMult = (1000 * durationMult).reduced();
+            const int noteLen = durationMult.numerator() / durationMult.denominator();
+
+            NoteEventList nel;
+            NoteEvent ne;
+            ne.setLen(noteLen);
+            nel.append(ne);
```

Comment:
That method doesn't exist (anymore? The move from `QVector` to `std::vector`?, if so it'd be 'nel.push_back(ne)` here I guess).